src/Account.ts

```diff
@@ -80,7 +80,7 @@ export class Account<T = unknown> {
       throw new Error('failed to get info about accounts ' + unsafeRes.error.message);
     }
     if (!unsafeRes.result.value) return;
-    const infos = (unsafeRes.result.value as AccountInfo<string[]>[]).map((info) => ({
+    const infos = (unsafeRes.result.value as AccountInfo<string[]>[]).filter((info) => info != null).map((info) => ({
```

Comment:
what about strict comparison?

Reply:
```suggestion
    const infos = (unsafeRes.result.value as AccountInfo<string[]>[]).filter(Boolean).map((info) => ({
```

Reply:
Probably cleaner with the Boolean cast, thank you ! I didn't use a strict comparison because it should filter out undefined as well.